contrib/gmime/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "gmime"
+pkgver = "3.2.13"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--disable-dependency-tracking",
```

Comment:
this will probably become unneeded if you switch it to gmake

Reply:
true, sure

i wonder if it's possible to universally pass it? afaik gnu_conf always takes it, and it only speeds up configure since we don't care

Reply:
it's probably better not to, when the dependency tracking setup fails it's better to figure out why than to silence it because it often indicates problems which may or may not result in miscompilation